README.md

```diff
@@ -19,17 +19,21 @@ Getting and Building the Code
         - Microsoft Visual Studio
     - CMake
     - Python
+    
+- Optional:
+    - git
 
 #### 2. Download the hdRpr source code
 
 You can use ```git``` to clone the repository.
+For modern, SMT or multi-threaded systems use:
 
 ```
-> git clone --recurse-submodules -j2 https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD
+> git clone --recurse-submodules -j64 https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD
```

Comment:
I would change this to say "git clone --recurse-submodules -j<num_cores> https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD"  where <num_cores> is the number of CPU cores, etc.

Reply:
Reason being that 64 threads even though there are only 5 submodules would oversubscribe an 8 core system.

Reply:
Not really. Modern schedulers and well written software, like servo or Firefox nightly uses more threads than physical cores, too.

Reply:
Could someone post the jenkins log then I would investigate why it's failing.

Reply:
Your changes do not affect compilation or tests